lnwallet/interface_test.go

```diff
@@ -950,7 +987,7 @@ func testTransactionSubscriptions(miner *rpctest.Harness,
 	// implementation of the WalletController.
 	txClient, err := alice.SubscribeTransactions()
 	if err != nil {
-		t.Fatalf("unable to generate tx subscription: %v", err)
+		t.Skipf("unable to generate tx subscription: %v", err)
```

Comment:
Why `Skipf`? To skip running the test on implementations that haven't implemented `SubcribeTransactions`? Or is it just a typo? :p

Reply:
I was making it conform to the comment above this line of code, since I was making changes below anyway to handle the fact that neutrino doesn't implement unconfirmed transactions. But yes, it's for potential future implementations that don't implement `SubscribeTransactions`. I can change it back if desired, since `SubscribeTransactions` can be implemented with just confirmed transactions like neutrino does.